core/src/test/scala/restui/models/ContentTypeSpec.scala

```diff
@@ -0,0 +1,20 @@
+package restui.models
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+class ContentTypeSpec extends AnyFlatSpec with Matchers with TableDrivenPropertyChecks {
+  private val properties = Table(
+    ("file", "type"),
+    ("test.yaml", ContentType.Yaml),
+    ("test.yml", ContentType.Yaml),
+    ("test.json", ContentType.Json),
+    ("test.txt", ContentType.Plain)
```

Comment:
edge cases ? and what about if reads something else ?

Reply:
There are 3 types:

- Yaml: `yaml`,`yml`
- Json: `json`
- Others: whatever